exporter/loadbalancingexporter/resolver_static.go

```diff
@@ -53,7 +53,9 @@ func newStaticResolver(endpoints []string) (*staticResolver, error) {
 }
 
 func (r *staticResolver) start(ctx context.Context) error {
-	r.resolve(ctx) // right now, this can't fail
+	if _, err := r.resolve(ctx); err != nil {  // right now, this can't fail
```

Comment:
Lint fails here:
```
golangci-lint run --allow-parallel-runners
resolver_static.go:56: File is not `gofmt`-ed with `-s` (gofmt)
	if _, err := r.resolve(ctx); err != nil {  // right now, this can't fail
```